robottelo/ui/contentviews.py

```diff
@@ -722,3 +722,40 @@ def fetch_puppet_module(self, cv_name, module_name):
             else:
                 raise UINoSuchElementError(
                     "Couldn't find puppet-modules tab")
+
+    def copy_view(self, name, new_name=None):
+        """
+        Copies an existing Content View
+        """
+        cv = self.search(name)
+        if cv and new_name:
```

Comment:
`search` returns `None` if it fails to find an element. Performing an explicit comparison against `None` is more robust and will prevent subtle bugs.

    if cv is not None and new_name is not None: